utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -383,6 +399,268 @@ impl<'trie, T: TrieValue + Into<u32>> CodePointTrie<'trie, T> {
     pub fn get_u32(&self, code_point: u32) -> u32 {
         self.get(code_point).into()
     }
+
+    /// Returns a [`CodePointMapRange`] struct representing a range of code
```

Comment:
Nit: If we keep this function public, as Markus is suggesting, we need to put in more work to document it and add an example.  When I look at this function I really have very little idea what it does or how to use it.